client/src/components/profileInfoEdit/ProfileClearModal.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import toast from 'react-hot-toast';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { clearTeam, deleteGHPages } from '../../actions/team';
+
+const ProfileClearModal = ({ clearAlert, setclearAlert }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const HandleClear = () => {
+    const access_token = localStorage.getItem('GH_access_token');
+    if (access_token === null) {
+      toast.error('Log in with github account');
```

Comment:
Instead of doing it like this, if the user wants to delete their website/repo, it'd be nice to let them log in on this page and not have to go to the dashboard to login and then come back to the profile page. So you can take the same logic as the github button and use it in this flow.